### ISSUE TITLE
Clarify guidelines for video use

### ISSUE THREAD

**Issue by timfrick** [w3c/sustainableweb-ig#13]
This suggestion was given to me by an accessibility professional on LinkedIn regarding video:

> The immediate one I think of relates to video which is helpful for people who struggle with written text and as relay necessary for Sign language interpretation on calls. You are right that your guidance doesn't contradict, however I notice that it doesn't mention it either? 

2.16 does specifically mention accessibility benefits:

> Delivering media assets in ways that convey information in an easy-to-read manner both visually and contextually (even if people are unable to for example see), will allow a wider audience to gain from your content.

However, to her point, I'm a little hung up on the phrasing 'easy to read'. Shouldn't that also be easy to perceive, understand, etc. per WCAG? Are there opportunities to be more specific and clear here, especially regarding the use case she mentions?
